collaborative-groth16/src/groth16.rs

```diff
@@ -116,8 +116,20 @@ where
     pub witness: FieldShareVec<T, P>,
 }
 
+impl<T, P: Pairing> Clone for SharedWitness<T, P>
```

Comment:
a comment on why this is needed (deriving clone would add bounds on T,P as well which are not needed)

Reply:
done (in other PR)